plaidml/bridge/openvino/plaidml_builder.cpp

```diff
@@ -34,14 +34,21 @@ class ProgramBuilder {
 
   const InferenceEngine::ICNNNetwork& network;
 
+  // cache network input/output info
+  InferenceEngine::InputsDataMap networkInputs;
+  InferenceEngine::OutputsDataMap networkOutputs;
```

Comment:
It's better not to put state onto the class if it's only ever used in a single function.

Reply:
These are used in every call to `handleParameter` (for inputs) or `handleOutput` (for outputs); my impression was `get*putsInfo` were vaguely expensive functions, hence the caching, but I can certainly call those on each use instead.

Reply:
I see, I didn't realize they were used also in `handleParameter` until after I commented here.